core/test/functional/routes/frontend_spec.js

```diff
@@ -476,6 +491,20 @@ describe('Frontend Routing', function () {
                 .expect(200)
                 .end(doEnd(done));
         });
+
+        it.skip('/blog/welcome-to-ghost/amp/ should 302 when AMP is disabled', function (done) {
```

Comment:
I couldn't make this test work properly, but tested it manually in every way, I could think of (with and without subdirectory (-> with and without trailing slash), static pages, posts, with AMP enabled and disabled. Everything works fine for me in manual tests.